### PR TITLE
google-benchmark: 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by theopolis**

### CODE REVIEW

File: Library/Formula/google-benchmark.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class GoogleBenchmark < Formula
+  desc "A microbenchmark support library"
+  homepage "https://github.com/google/benchmark"
+  url "https://github.com/google/benchmark/archive/v0.1.0.tar.gz"
+  version "0.1.0"
```
Comment (bfontaine) [inline Library/Formula/google-benchmark.rb]: You can remove this line.

File: Library/Formula/google-benchmark.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class GoogleBenchmark < Formula
+  desc "A microbenchmark support library"
```
Comment (bfontaine) [inline Library/Formula/google-benchmark.rb]: Please remove `A ` here.

File: Library/Formula/google-benchmark.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+
+    system "cmake", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
```
Comment (bfontaine) [inline Library/Formula/google-benchmark.rb]: Please add a test.

**Comment by theopolis** [comment]
Sorry! It was late and I forgot to `brew audit`. Should be passing now.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/d36053bf87bb49b44ed2b7471f63d6b207ab802a. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just a nit, but new formula & version bumps don't need the `:` separator in the commit message.

**Pull request closed without merging**